pyomo/contrib/simplification/tests/test_simplification.py

```diff
@@ -0,0 +1,117 @@
+from pyomo.common.unittest import TestCase
```

Comment:
Can you please add the copyright statement?

Reply:
Done.